### ISSUE TITLE
Build fails on current nightly (1.74.0-nightly), due to known proc-macro2 issue

### ISSUE THREAD

**Issue by 64kramsystem** [hardpixel/mrml-ruby#6]
On the current nightly (1.74.0-nightly), compilation fails with this error:

```
error[E0635]: unknown feature `proc_macro_span_shrink`
  --> /home/saverio/.cargo/registry/src/index.crates.io-6f17d22bba15001f/proc-macro2-1.0.56/src/lib.rs:92:30
   |
92 |     feature(proc_macro_span, proc_macro_span_shrink)
   |                              ^^^^^^^^^^^^^^^^^^^^^^
```

This issue is [known](https://github.com/dtolnay/proc-macro2/issues/398) and it has been solved in `proc-macro2` v1.0.60, so I think the project should update the dependency.

**Comment by 64kramsystem** [comment]
I've noticed that this has been fixed in `mrml-ruby`'s master (9e0e75e8faf11814c34c6f2e108b868cca3f57ea); it'd be helpful if a new version with that commit was released :grimacing:

**Comment by jonian** [comment]
Hi @64kramsystem, thanks for the report. I have released a new version.
